src/pykeen/triples/triples_factory.py

```diff
@@ -418,15 +418,23 @@ def split(
             training_factory, testing_factory, validation_factory = factory.split(ratios)
         """
         triples_groups = split(
-            triples=self.triples,
+            # TODO: work around
+            triples=self.mapped_triples.numpy(),
             ratios=ratios,
             random_state=random_state,
             randomize_cleanup=randomize_cleanup,
             method=method,
         )
         return [
             TriplesFactory(
-                triples=triples,
+                triples=np.asarray([
+                    (
+                        self.entity_id_to_label[hi],
+                        self.relation_id_to_label[ri],
+                        self.entity_id_to_label[ti],
+                    )
+                    for hi, ri, ti in triples
+                ]),
```

Comment:
is this enough workaround that we can merge this PR before all of the other triples factory stuff?

Reply:
It will likely be slower than status quo

Reply:
Correctness should be unchanged.